website/views/static/api.jade

```diff
@@ -86,8 +86,9 @@ html
         li POST to the URL <code>/api/v2/user/tasks/{id}/{direction}</code>
           ul
            li <code>{direction}</code> is 'up' or 'down'
-           li <code>{id}</code> is a unique identifier for a task, which you make up, consisting of lowercase letters. Try to make it something common, like 'productivity' or 'fitness' - because other services may piggy-back off <em>your</em> task. For example, the Chrome extension down-scores a <code>productivity</code> task when you visit vice websites (reddit, 9gag, etc). However, Pomodoro up-scores <code>productivity</code> when you complete a task. So the two services share a single task to score your overall productivity. If the task doesn't yet exist, it is created the first time you POST to this URL.
+           li <code>{id}</code> is a unique identifier for a Habit, which you make up, consisting of lowercase letters. Try to make it something common, like 'productivity' or 'fitness' - because other services may piggy-back off <em>your</em> Habit. For example, the Chrome extension down-scores a <code>productivity</code> Habit when you visit vice websites (reddit, 9gag, etc). However, Pomodoro up-scores <code>productivity</code> when you complete a Pomodoro task. So the two services share a single Habit to score your overall productivity. If the Habit doesn't yet exist, it is created the first time you POST to this URL.
```

Comment:
There is a link to the wiki's Extension category. I believe the Chrome Extension is still a bit buggy so I don't want to push it much.

The API doesn't downcase IDs.